department-of-veterans-affairs/caseflow
# TYLERB/APPEALS-19620: Prevent non logged in users from viewing membership request form

Resolves [APPEALS-19620](https://vajira.max.gov/browse/APPEALS-19620)

### Description
Added a system check to prevent the user from seeing and user the VhaMembershipRequestForm unless they have logged in before viewing the page.

### Acceptance Criteria

### Testing Plan
This testing plan will not work on demo or development. One way to test it locally would be to change the user model method authenticated? to always return false and then view the page. It might break other things though.
1. Visit the url /vha/help
2. Verify that the form is not displayed without logging in


### User Facing Changes

### Storybook Story
*For Frontend (Presentationa) Components*
